src/services/news2/uc/news.facade.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Codacy found an issue: [Too many blank lines at the beginning of file. Max of 0 allowed.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045784)